lib/mixins/template-stamp.html

```diff
@@ -313,6 +315,32 @@
        *   metadata to `nodeInfo`
        */
       static _parseTemplateNestedTemplate(node, outerTemplateInfo, nodeInfo) {
+        // If a `<dom-repeat>` or a `<dom-if>` is inside of an svg, we need
+        // to create all elements in a different namespace. To do so, make an svg,
+        // let the HTML parser parse in that namespace, and then put back all
+        // created children in a fresh template.
+        if (node.parentNode.localName && templateExtensions[node.parentNode.localName]
+              && node.parentNode.parentNode && node.parentNode.parentNode.namespaceURI === SVG_NAMESPACE) {
+          const svg = document.createElementNS(SVG_NAMESPACE, 'svg');
+          svg.innerHTML = node.innerHTML;
+
+          const template = /** @type {HTMLTemplateElement} */ (document.createElement('template'));
```

Comment:
Rather than creating a new template and replacing it, I think it's best to replace the contents of the existing template:

```js
node.textContent = '';
while (svg.firstChild) {
  node.content.appendChild(svg.firstChild);
}
```

Reply:
I tried this at first, but this node does not have content. In other words, it is not actually a template. I think this happens because it parses template in the svg namespace, and this element does not exist there. Therefore I needed to create a new template to have the correct documentfragment to work with.

Reply:
Oh yeah, but we want to make a `Polymer.svg` helper that returns a template and use that here.  So it would be:

```js
const svgTemplate = Polymer.svg([node.innerHTML]);
node.textContent = '';
node.appendChild(svgTemplate.content);
```
(and we throw away the `svgTemplate`; if this was a hot path we'd care about the extra template node but it's not really)

Reply:
But `node.content` is undefined in this context. You cant assign it or use it here.